chapters/synchronous.tex

```diff
@@ -977,7 +991,7 @@ \subsection{Base-Clock Partitioning}\label{base-clock-partitioning}
 0 = f6(y, x1, u);
 \end{lstlisting}
 
-After base-clock partitioning, the following partitions are identified:
+After base-partitioning, the following partitions are identified:
 \begin{lstlisting}[language=modelica]
 // Base partition 1 -- clocked partition
```

Comment:
We say _clock partition_ now, right?
```suggestion
// Base partition 1 -- clock partition
```

Reply:
If not, I suppose we should say _clocked partition_ everywhere instead?

Reply:
I don't see the issue. The 1st Base partition is a "Clocked Partition"; instead of "Continuous-Time".